src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -492,12 +492,50 @@ internal static FamilyInstance FromExisting(Autodesk.Revit.DB.FamilyInstance fam
 
         #endregion
 
+        #region Public Methods
+
         public override string ToString()
         {
             return InternalFamilyInstance.IsValidObject ? InternalFamilyInstance.Name : "empty";
         }
 
-        #region Public methods
+        /// <summary>
+        /// Get FamilyInstance Host Element
+        /// </summary>
+        public Element GetHost
+        {
+            get
+            {
+                if (this.InternalFamilyInstance.Host != null)
+                    return ElementWrapper.Wrap(this.InternalFamilyInstance.Host, true);
```

Comment:
if this is not feasible to solve quickly, do not worry about it. But please add a comment / TODO and I will add a task to followup.